assets/src/infrastructure/ui/inputs/dateTime/Timepicker/style.scss

```diff
@@ -1,26 +1,158 @@
+}
 
-		> div {
-			background-color: transparent;
-			padding: 0 0.5rem;
+.ee-time-picker-focus {
+	border-color: var(--ee-color-primary);
+	/* Visible in Windows high-contrast themes */
+	outline-color: transparent;
+	outline-width: 3px;
+	outline-style: solid;
 
-			@extend .ee-base-inputs-styles;
-			&:hover {
-				@extend .ee-input-base-hover;
-			}
-			&:focus {
-				@extend .ee-input-base-focus;
+	&:active {
+		border-color: var(--ee-color-primary-high-contrast);
+	}
+}
+
+.ee-time-picker-hover {
+	border-color: var(--ee-color-primary-low-contrast);
+
+	&:active {
+		border-color: var(--ee-color-primary-high-contrast);
+	}
+}
+
+.ee-input-base-wrapper {
+	.react-time-picker {
+		.react-time-picker {
+			&__inputGroup {
+				border-color: var(--ee-border-color);
+				border-radius: var(--ee-border-radius-small);
+				border-style: solid;
+				border-width: 3px;
+				box-shadow: none;
+				box-sizing: border-box;
+				color: var(--ee-default-text-color);
+				line-height: inherit;
+				height: 2.625rem;
+				margin: 0 var(--ee-margin-nano) 0 0;
+				outline: none;
+				padding: var(--ee-padding-nano) var(--ee-padding-smaller);
+				transition: all ease 50ms;
+
+				&__input,
+				&__leadingZero {
+					border: none !important;
+					border-radius: unset !important;
+					box-shadow: none !important;
+					color: var(--ee-default-text-color);
+					display: inline-block;
+					font-size: var(--ee-font-size-default);
+					line-height: inherit;
+					min-height: unset !important;
+					outline: none !important;
+					margin: 0;
+					padding: unset !important;
+					transition: unset !important;
+				}
+
+				&__amPm {
+					-webkit-appearance: auto;
+					background: unset;
+					vertical-align: unset;
+
+					option {
+						background-color: var(--ee-background-color) !important;
+						color: var(--ee-default-text-color) !important;
+						height: var(--ee-icon-button-size) !important;
+					}
+
+					&:focus {
+						background-color: var(--ee-color-primary-low-contrast);
+						color: var(--ee-text-on-primary);
+						height: 22.5px;
+					}
+				}
+
+				&__divider {
+					color: var(--ee-color-grey-10);
+					font-size: var(--ee-font-size-big);
+					font-weight: 400;
+					margin: 0 var(--ee-margin-micro);
+					padding: 0;
+				}
+
+				// the following style rules do the same thing, but need to be kept separate, else they no workie
+				input[type='number']::selection {
+					background-color: var(--ee-color-primary-low-contrast);
+					color: var(--ee-text-on-primary);
+				}
+				input[type='number']::-moz-selection {
+					background-color: var(--ee-color-primary-low-contrast);
+					color: var(--ee-text-on-primary);
+				}
+				select::selection {
+					background-color: var(--ee-color-primary-low-contrast);
+					color: var(--ee-text-on-primary);
+				}
+				select::-moz-selection {
+					background-color: var(--ee-color-primary-low-contrast);
+					color: var(--ee-text-on-primary);
+				}
```

Comment:
![image](https://user-images.githubusercontent.com/1477453/87020149-22db8880-c1dc-11ea-92ba-ba4560b88ab4.png)
 

```suggestion
input[type='number']::selection,
input[type='number']::-moz-selection,
select::selection,
select::-moz-selection {
  background-color: red;
	color: green;
}
				 

```

Reply:
doesn't work that way, that's why I added this comment:

```
the following style rules do the same thing, but need to be kept separate, else they no workie
```

see this article for an explanation (near the end of the article):
https://css-tricks.com/overriding-the-default-text-selection-color-with-css/

Reply:
oh, I see it's related to those prefixes, later we can improve this with some mixins:
https://css-tricks.com/snippets/sass/mixin-prefix-properties/
***
not related to this PR